migrations/0102.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE stock_log
+ADD qu_factor_purchase_to_stock REAL NOT NULL DEFAULT 1;
```

Comment:
Adding a column with a `NOT NULL` constraint to a table with existing data is not possible in SQLite...

Reply:
Hmm, there were no sql errors and it looks to have been added in my dev db?
I did not test adding a null yet though.

Reply:
Yes, sorry, seems to work - did only a quick test via my preferred SQLite editor where it did not work, but seems to work when executed through PHP (whatever the difference is)...